src/middleware/permissionsMiddleware.js

```diff
@@ -56,7 +56,8 @@ const permissions = shield({
     CreateBadge: isAdmin,
     UpdateBadge: isAdmin,
     DeleteBadge: isAdmin,
-
+    addSocialMediaAccount: isAuthenticated,
```

Comment:
Is it really social media when you can enter everything? Should we maybe limit that can be entered to prevent abuse? What do you think?